src/main/java/org/jabref/gui/preferences/GeneralTabViewModel.java

```diff
@@ -78,6 +79,7 @@ public GeneralTabViewModel(DialogService dialogService, JabRefPreferences prefer
         );
     }
 
+    @Override
```

Comment:
no need to override for setValues(), storeSettings() or validateSettings(), they directly implement the PrefsTab-Interface

Reply:
Oh sorry, this is because of the Eclipse IDE that put this automatically. I will correct this.